packages/html/src/events/form.rs

```diff
@@ -12,6 +20,8 @@ pub struct FormData {
 
     pub values: HashMap<String, Vec<String>>,
 
+    pub value_types: HashMap<String, String>,
```

Comment:
Instead of having two different maps for the values and the value types, can we use one hashmap of `HashMap<String, ValueType>`? That will let different renderers decide what events have multiple values and it insures that every value has a value_type

Reply:
That will be neat. But in that case, we might need to check the effect. I'll make the changes and let you know 👾

Reply:
I also wanted to ask if this code can be moved as a common util function:

```rust
fn convert_hashmap_to_json<K, V>(hashmap: &HashMap<K, V>) -> serde_json::Result<String>
where
    K: Serialize + std::hash::Hash + Eq,
    V: Serialize,
{
    serde_json::to_string(hashmap)
}

```

Reply:
I don't feel like that will come up again. We don't have a common crate that is shared between all of the Dioxus crates. If we need to deserialize from a hashmap into another structure we can just copy-paste the function or reconsider and pull out the function

Reply:
> Instead of having two different maps for the values and the value types, can we use one hashmap of `HashMap<String, ValueType>`? That will let different renderers decide what events have multiple values and it insures that every value has a value_type

Do you think we can make it a separate PR? Because it seems like a new feature. I would gladly work on that as well 😇.

Reply:
You can open a separate PR if you would prefer that. It is just a different way to structure the data, not necessarily a new feature. I'm also happy to take over implementing that change if it isn't something you are interested in.

We are preparing for a 0.5 release soon. Because this changes the public API of `FormData`, I don't think we should merge this yet if we know the API will change soon so that we can release 0.5 before merging the followup PR.

Reply:
Sure, I understand. I have already started working on the requirements, We will work this out together. 

Really excited for 0.5 🧬🧬🧬.  I'm just glad that I'm part of it (I am right ? 🫣)